src/Stratis.Bitcoin/Configuration/Settings/ConnectionManagerSettings.cs

```diff
@@ -218,5 +219,8 @@ public static void PrintHelp(Network network)
 
         /// <summary><c>true</c> to enable bandwidth saving setting to send and received confirmed blocks only.</summary>
         public bool RelayTxes { get; set; }
+
+        /// <summary>Disable functionality whereby nodes disallow connection to peers that are within the same IP range to prevent sybil attacks.</summary>
+        public bool? IpRangeFiltering { get; internal set; }
```

Comment:
You should add remarks section explaining the exact ranges for each version of IP protocol

Reply:
Have commented method PeerIsPartOfExistingGroup but do I misunderstand the level of detail?  Groups for Type C IP addresses just compare the first XXX.XXX